gradle.properties

```diff
@@ -32,7 +32,7 @@ spring_no_http_plugin_version=0.0.3.RELEASE
 checkstyle_version=8.24
 openapi_plugin_version=4.2.2
 
-xm_commons_version=2.1.3
+xm_commons_version=2.1.2
```

Comment:
https://mvnrepository.com/artifact/com.icthh.xm.commons/xm-commons-permission/2.1.3
2.1.3 already in maven )

Reply:
Done